addons/udes_priorities/tests/test_stock_picking.py

```diff
@@ -23,3 +23,31 @@ def test_correct_picking_type_or_no_picking_type_is_valid(self):
 
         self.assertEqual(self.not_urgent.picking_type_ids, PickingType.browse())
         Picking.create({"priority": self.not_urgent.reference})
+
+    def test_priority_after_adding_move_to_picking(self):
+        Picking = self.env["stock.picking"].with_context(
+            default_picking_type_id=self.picking_type_pick.id
+        )
+        Product = self.env["product.product"]
+        Move = self.env["stock.move"]
+        apple = Product.create(
+            {"name": "Apple", "barcode": "bar-apple", "default_code": "APL", "type": "product",}
+        )
+        self.not_urgent.picking_type_ids = self.picking_type_pick
+        self.assertEqual(self.not_urgent.picking_type_ids, self.picking_type_pick)
+        pick = Picking.create({"priority": self.not_urgent.reference})
+        self.assertEqual(pick.priority, self.not_urgent.reference)
+        move = Move.create(
+            {
```

Comment:
as you see here we provide nothing for the priority for the move and so it is defaulted to normal, which means we must include that in the selection domain defined by the method "get_priorities_for_selection"